src/volue/mesh/_base_session.py

```diff
@@ -845,6 +845,40 @@ def run_simulation(
             grpc.RpcError
         """
 
+    @abc.abstractmethod
+    def run_inflow_calculation(
+        self,
+        model: str,
+        area: str,
+        water_course: str,
+        start_time: datetime,
+        end_time: datetime,
+    ) -> Union[typing.Iterator[None], typing.AsyncIterator[None]]:
+        """Run an inflow calculation using HydSim on the Mesh server.
+
+        Args:
+            model: The name of the Mesh model in which to run the inflow
+                calculation exists.
```

Comment:
```suggestion
            model: The name of the Mesh model in which the inflow
                calculation exists.
```

Not sure if the `calculation` word could be move one line up.

Reply:
Thanks, fixed :smile: